apps/website/src/server/trpc/router/donations.ts

```diff
@@ -63,10 +63,12 @@ export const donationsRouter = router({
       return createDonations(input.donations);
     }),
 
-  getPixels: sharedKeyProcedure.query(async () => {
-    const pixels = await getPixels();
-    return pixels;
-  }),
+  getPixels: sharedKeyProcedure
+    .input(z.object({ muralId: z.string() }))
+    .query(async ({ input }) => {
+      const pixels = await getPixels(input.muralId);
+      return pixels;
+    }),
```

Comment:
This is such a tiny nit and I am not even sure whats better, but we can save one Promise wrap and some characters by doing this. Not sure why it bothers me ;)

```suggestion
    .query({ input }) => getPixels(input.muralId)),
```

Reply:
Oops missed a ( there